Apps/Snake/tactility.py

```
@@ -0,0 +1,693 @@
+def package_all(platforms):
+    status = f"Building package with {platforms}"
+    print_status_busy(status)
+    package_intermediate(platforms)
+    # Create build/something.app
+    try:
+        tar_path = package_name(platforms)
+        tar = tarfile.open(tar_path, mode="w", format=tarfile.USTAR_FORMAT)
+        tar.add(os.path.join("build", "package-intermediate"), arcname="")
+        tar.close()
+        print_status_success(status)
+        return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)